packages/dts-gen/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "dts-gen",
+  "version": "0.7.3",
+  "description": "TypeScript Definition File Generator",
+  "author": "Ryan Cavanaugh <ryanca@microsoft.com> (https://www.github.com/RyanCavanaugh)",
+  "homepage": "https://github.com/microsoft/DefinitelyTyped-tools/tree/main/packages/typescript-packages#readme",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "bin": {
+    "dts-gen": "dist/run.js"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/microsoft/DefinitelyTyped-tools.git",
+    "directory": "packages/dts-gen"
+  },
+  "bugs": {
+    "url": "https://github.com/microsoft/DefinitelyTyped-tools/issues"
+  },
+  "engines": {
+    "node": ">=16.17.0"
+  },
+  "dependencies": {
+    "dts-dom": "^3.7.0",
+    "parse-git-config": "^1.1.1",
+    "typescript": "^3.5.1",
```

Comment:
This is ancient. I haven't modified this but I am definitely curious if we can safely bump this.

Reply:
It's a real dep; it uses TS to parse and walk ASTs and so on. But I suspect that this being ancient means that it's not been working super well.

Reply:
Bumped to 5.3.3. FWIW the repo in general is still on 5.2.2, which we should update.